django-rest/imdb_app/imdb_api/views.py

```diff
@@ -1,3 +1,34 @@
 from django.shortcuts import render
+from rest_framework import permissions, viewsets
+from .models import Movie, Genre, Comment
+from .serializers import MovieSerializer, GenreSerializer, CommentSerializer
+from django_filters.rest_framework import DjangoFilterBackend
 
-# Create your views here.
+class MovieViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
```

Comment:
```suggestion
    API endpoint that allows movie to be viewed or edited.
```

Jel ti treba da moze bilo ko da edituje movie? Zar nije samo view da ti treba? Bez edita, ovako moze bilo da edituje

Reply:
U specifikaciji postoji Movie Edit Screen feature, pa zato